File: data/python/calculadora.py
#Declaramos variables para el calculo de una calculadora

def funcion_suma(a,b):
    resultado = a+b
    return resultado

def funcion_resta(a,b):
    resultado = a-b
    return resultado

def funcion_multiplicacion(a,b):
    resultado = a*b
    return resultado

def funcion_division(a,b):
    resultado = a/b
    return resultado

print ("1.- Operacion Suma")
print ("2.- Operacion Resta")
print ("3.- Operacion Multiplicacion")
print ("4.- Operacion Division")

opcion_elegida = input ("Selecciona una opcion de las siguientes: ")

op = int (opcion_elegida)
print (op)

if op == 1:
    print("\nOperacion Suma elegida. Introduzca los dos numeros\n")
    numero1 = input ("Numero 1: ")
    numero2 = input ("\nNumero 2: ")
    numero11 = int(numero1)
    numero22 = int(numero2)
    int(numero2)
    resultado_op = funcion_suma (numero11,numero22)
    print ("El resultado es: ",resultado_op)
    
elif op == 2 :
    print("Operacion Resta elegida. Introduzca los dos numeros")
    numero1 = input ("Numero 1: ")
    numero2 = input ("Numero 2: ")
    numero11 = int(numero1)
    numero22 = int(numero2)
    resultado_op = funcion_resta (numero11,numero22)
    print ("El resultado es: ",resultado_op)
    
elif op == 3 :
    print("Operacion Multiplicacion elegida. Introduzca los dos numeros")
    numero1 = input ("Numero 1: ")
    numero2 = input ("Numero 2: ")
    numero11 = int(numero1)
    numero22 = int(numero2)
    resultado_op = funcion_multiplicacion (numero11,numero22)
    print ("El resultado es: ",resultado_op)
    
elif op == 4 :
    print("Operacion Division elegida. Introduzca los dos numeros")
    numero1 = input ("Numero 1: ")
    numero2 = input ("Numero 2: ")
    numero11 = int(numero1)
    numero22 = int(numero2)
    resultado_op = funcion_division (numero11,numero22)
    print ("El resultado es: ",resultado_op)

File: data/c/calculadora.c
/*
  25065787 jhonny jose arana nieves <jhonnyjosearana@gmail.com>
 */
#include <stdio.h>
/* funcion para sumar */
double suma(double x,double y){
	double resSuma;
		resSuma = x + y;
	return resSuma;
	}
/* funcion para restar*/
	double resta(double x,double y){
	double resResta;
		resResta = x - y;
	return resResta ;
	}
/* funcion para multiplicar */
	double multiplicacion(double x,double y){
	double resMulti;
		resMulti= x * y;
	return resMulti;
	}
/* funcion para dividir */
	double division(double x,double y){
	double resDivi;
		resDivi = x / y;
	return resDivi;
	}
/* funcion que muestra el menu y retorna la opcion selecionada */
int menu(){
	int opcion;
	printf("\nque operacion desea realizar.? ");
	printf("\n1  - - suma");
	printf("\n2 - - resta");
	printf("\n3 - - multiplicacion");
	printf("\n4 - -division\n\n");
	scanf("%d",&opcion);
	return opcion;
}
/* funcion main compara la opcion seleccionada y los numeros 
   introducidos llama a las funciones respectivas
   e imprime el resultado */
int main(){
	float resultado;
	int a;
	int  b;
	int opcionmenu;
	printf("\nintroduzca el primer numero:\t");
	scanf("%d",&a);
	printf("\nintroduzca el segundo numero:\t");
	scanf("%d",&b);
/* obtiene la opcion selecionada e invoca las otras funciones */	
	opcionmenu = menu(); 
		if(opcionmenu == 1){
			resultado = suma(a,b);
		} else if(opcionmenu == 2) {
			resultado= resta(a,b);
		} else if(opcionmenu == 3) {	
			resultado = multiplicacion(a,b);
		} else if(opcionmenu == 4){
			resultado = division(a,b);
		}
/* imprime el resultado */
	printf("\nel resultado es: %f",resultado);
	return 0;
}

/*
  25065787 jhonny jose arana nieves <jhonnyjosearana@gmail.com>
 */
#include <stdio.h>
/*funcion celsius a kelvin*/
double kelvin(double x){
	double resKelvin;
		resKelvin = x + 273.15 ;
	return resKelvin;
	}
/*funcion celsius a fahrenheit*/
double fahrenheit(double x){
	double resFah;
		resFah = x * 9/5 + 32;
	return resFah ;
	}
/*menu de opciones*/
int menu(){
	int opcion;
	printf("\ncomo desea hacer la conversion ");
	printf("\n1  - - celsius a kelvin\n");
	printf("\n2 - - celcius a fahrenheit\n\n");
	scanf("%d",&opcion);
	return opcion;
}
/*funcion main obtiene y retorna datos*/
int main(){
	float resultado;
	int a;
	int opcionmenu;
	printf("\nintroduzca el valor en grados celcius:\t");
	scanf("%d",&a);
	opcionmenu = menu(); 
		if(opcionmenu == 1){
			resultado = kelvin(a);
		} else if(opcionmenu == 2) {
			resultado= fahrenheit(a);
		} 
/* imprime el resultado */
	printf("\nel resultado es: %f",resultado);
	return 0;
}